actions/workflows/st2_package_test.yaml

```diff
@@ -0,0 +1,83 @@
+---
+version: '2.0'
+
+st2cd.st2_package_test:
+    type: direct
+    input:
+        - hostname
+        - dns_zone
+        - instance_type
+        - environment
+        - key_name
+        - keyfile
+        - distro
+        - role
+        - release
+        - version
+        - bootstrap_branch
+        - bootstrap_script
+        - debug
+        - st2_username: test
```

Comment:
`admin`, not `test` and BTW they are script defaults anyways why supplying extra defaults here?